src/shared/utils/get-active-tab-id.ts

```diff
@@ -0,0 +1,5 @@
+export function getActiveTab(): Promise<chrome.tabs.Tab[]> {
```

Comment:
I think this naming here (file vs fn) is slightly confusing bc it is used to get all tabs in some places, but in other is picks off one tab for the id?

Reply:
yeah so it's just weird it returns an array, even though i think there can only be 1 active. Maybe i should just resolve the first index within the promise?

Reply:
changed